### PR TITLE
move tempelis postsubmit to k8s infra

### PR DESCRIPTION
**Pull request by BenTheElder**
TODO: the presubmit

### CODE REVIEW

File: config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-tempelis.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  kubernetes/community:
+  - name: post-community-tempelis-apply
+    cluster: k8s-infra-prow-build-trusted
+    decorate: true
+    branches:
+    - ^master$
+    run_if_changed: '^communication/slack-config'
+    annotations:
+      testgrid-num-failures-to-alert: "1"
+      testgrid-alert-email: ktbry@google.com, k8s-infra-oncall@google.com
```
Comment (ameukam) [inline config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-tempelis.yaml:11]: (non-blocking comment): drop or:
```suggestion
      testgrid-alert-email: sig-contribex-leads@kubernetes.io
```
?

Comment (jberkus) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-tempelis.yaml:11]: feels like we should have a group email for the slack admins

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-tempelis.yaml]: moved to contribex leads with a TODO about slack admins list

**Pull request merged**